README.md

```diff
@@ -98,13 +98,13 @@ without TLS.**
 
 ### Accessing the services
 
-| Service        | URL                              | Credentials       |
-|----------------|----------------------------------|-------------------|
-| ORT Server API | http://localhost:8080/swagger-ui |                   |
-| Keycloak       | http://localhost:8081            | admin:admin       |
-| PostgreSQL     | http://localhost:5433            | postgres:postgres |
-| RabbitMQ       | http://localhost:15672           | admin:admin       |
-| Graphite       | http://localhost:8888            | root:root         |
+| Service        | URL                              | Credentials                                             |
+|----------------|----------------------------------|---------------------------------------------------------|
+| ORT Server API | http://localhost:8080/swagger-ui |                                                         |
+| Keycloak       | http://localhost:8081            | Administrator: admin:admin<br/>User: ort-admin:password |
```

Comment:
This change should be in a separate commit.

Reply:
Done.